Kudu.Services/Jobs/JobsController.cs

```diff
@@ -182,7 +182,8 @@ public HttpResponseMessage InvokeTriggeredJob(string jobName, string arguments =
 
                 // Return a 200 in the ARM case, otherwise a 202 can cause it to poll on /run, which we don't support
                 // For non-ARM, stay with the 202 to reduce potential impact of change
-                return Request.CreateResponse(ArmUtils.IsArmRequest(Request) ? HttpStatusCode.OK : HttpStatusCode.Accepted);
+                return Request.CreateResponse(ArmUtils.IsArmRequest(Request) ? HttpStatusCode.OK : HttpStatusCode.Accepted,
+                    ArmUtils.AddEnvelopeOnArmRequest(_triggeredJobsManager.GetLatestJobRun(jobName), Request));
```

Comment:
Get latest run is still approximate and could race with other triggers.   A more precise fix would be for this [API](https://github.com/projectkudu/kudu/blob/a6261ff068546cb9c1fd7c02e4fdb8f80d7fe338/Kudu.Core/Jobs/TriggeredJobRunner.cs#L44) to return logger.Id.   This id gets bubbled up.   It should not be many file changes I think.